src/components/ProposalForm.tsx

```diff
@@ -91,13 +151,22 @@ export default function ProposalForm() {
                 id="description"
                 placeholder="Provide more details about your proposal"
                 required
+                name="description"
+                minLength={6}
+                rows={5}
+                maxLength={200}
               />
             </div>
             <div className="flex justify-end">
-              <Button className="mr-2" type="submit" variant="outline">
+              <Button
+                className="mr-2"
+                type="submit"
+                name="status-draft"
+                variant="outline"
+              >
                 Save Draft
               </Button>
```

Comment:
TODO: Let's also comment out this button, at least until we get it working. 
```suggestion
//              <Button
//                className="mr-2"
//                type="submit"
//                name="status-draft"
//                variant="outline"
//              >
//                Save Draft
//              </Button>
```